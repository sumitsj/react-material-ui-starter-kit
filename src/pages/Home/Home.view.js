import React from "react";
import Typography from "@material-ui/core/Typography";

const Home = () => (
  <Typography variant="h6" style={{backgroundColor: 'cyan'}}>
    Home Page
  </Typography>
);


export default Home;
