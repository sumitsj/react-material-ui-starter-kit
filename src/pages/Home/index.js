import Home from './Home.view';

export default Home;
