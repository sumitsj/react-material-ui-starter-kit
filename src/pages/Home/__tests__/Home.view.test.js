import React from 'react';
import { render } from '@testing-library/react';
import Home from '../Home.view';

describe('Home', () => {
  describe('HTML Structure', () => {
    it('should render properly', () => {
      const { asFragment } = render(<Home />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
