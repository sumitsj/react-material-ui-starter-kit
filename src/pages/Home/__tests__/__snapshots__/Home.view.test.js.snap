// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home HTML Structure should render properly 1`] = `
<DocumentFragment>
  <h2
    class="MuiTypography-root MuiTypography-h2 MuiTypography-gutterBottom"
  >
    Home Page
  </h2>
</DocumentFragment>
`;
